cdap-ui/app/features/admin/routes.js

```diff
@@ -143,6 +143,34 @@ angular.module(PKG.name + '.feature.admin')
               templateUrl: '/assets/features/admin/templates/namespace/datasets.html',
               controller: 'AdminDatasetsController'
             })
+              .state('admin.namespace.detail.datasets.streamcreate', {
```

Comment:
Shouldn't this be ```admin.namespace.detail.streams.create```?

I believe we either display both streams & datasets in admin view under 'data' or we have individual streams & datasets submenus under namespace to show them individually. Right?